libraries/AR_Motors/AP_MotorsUGV.h

```diff
@@ -6,6 +6,15 @@
 
 class AP_MotorsUGV {
 public:
+    enum UnlockState {
+        UNLOCK_WAITING = 0,
+        UNLOCK_STEP_DEC_TO_MAX_NEG = 1,
+        UNLOCK_STEP_STAY_MAX = 2,
+        UNLOCK_STEP_INC_TO_ZERO = 3,
+        UNLOCK_STEP_ZEROS_2 = 4,
+        UNLOCK_FINISHED = 5
+    };
```

Comment:
I think this can be private.

Reply:
sorry, should have been clearer, better to put this down with the other private stuff idealy with the varable that uses is. Ie:
```
    enum UnlockState {
        UNLOCK_WAITING = 0,
        UNLOCK_STEP_DEC_TO_MAX_NEG = 1,
        UNLOCK_STEP_STAY_MAX = 2,
        UNLOCK_STEP_INC_TO_ZERO = 3,
        UNLOCK_STEP_ZEROS_2 = 4,
        UNLOCK_FINISHED = 5
    } _esc_backward_unlock_state ; // state machine status for backward unlock logic.
```

Reply:
@HefnySco